src/pspm_options.m

```diff
@@ -58,12 +58,15 @@
   case 'convert_area2diameter'
     % 2.6 pspm_convert_area2diameter --
     options = autofill_channel_action(options);
+    options = autofill(options, 'channel',                'both',      '*Int*Char'       );
```

Comment:
My understanding is, unless for this case ('both' means 'left' and 'right'), `channel` should be just referring to one channel, either a number or the name of a specific channel. Is this the expected behaviour?